sympy/physics/tests/test_units.py

```diff
@@ -20,6 +20,8 @@ def test_units():
     t = Symbol('t')
     assert integrate(t*m/s, (t, 1*s, 5*s)) == 12*m*s
     assert (t * m/s).integrate((t, 1*s, 5*s)) == 12*m*s
+    
+    assert float( (year / week).evalf(15) ) == 52.17745571428571
```

Comment:
Yes, you are correct. Updated. Note that I also imported `Float`.

Thank you for teaching me about the subtleties of testing using SymPy and Python types.